docker/images/n8n/README.md

```diff
@@ -212,18 +217,18 @@ the environment variable `TZ`.
 
 Example to use the same timezone for both:
 
-```
+```bash
 docker run -it --rm \
-	--name n8n \
-	-p 5678:5678 \
-	-e GENERIC_TIMEZONE="Europe/Berlin" \
-	-e TZ="Europe/Berlin" \
-	n8nio/n8n
+ --name n8n \
+ -p 5678:5678 \
+ -e GENERIC_TIMEZONE="Europe/Berlin" \
+ -e TZ="Europe/Berlin" \
+ docker.n8n.io/n8nio/n8n
 ```
 
 ## Build Docker-Image
 
-```
+```bash
 docker buildx build --platform linux/amd64,linux/arm64,linux/arm/v7 --build-arg N8N_VERSION=<VERSION> -t n8nio/n8n:<VERSION> .
```

Comment:
should we change this from `n8nio/n8n:<VERSION>` to `n8n:<VERSION>`. no one building an image should be using the org-name in the tag.